### DISCUSSION TITLE
extremely low probability inference on pretrained resnet50-v1-12.onnx model

### DISCUSSION THREAD

**Discussion by Apisteftos** [onnx/onnx#5106] [Q&A]
I am trying actually to quantize the pretrained resnet50 model  and I am getting a low prediction, I have my own preprocessing model but I tried to compared with the provided one. 

[onnx preprocessing resnet50](https://github.com/onnx/models/blob/main/vision/classification/onnxrt_inference.ipynb)
[onnx preprocessing efficientnet-lite](https://github.com/onnx/tensorflow-onnx/blob/main/tutorials/efficientnet-lite.ipynb)

here is the outcome with `kitten.jpg`and `panda.jpg` according to provided preprocessing adjustments.  

`class=n02123159 tiger cat ; probability=12.294151` 
`class=n02510455 giant panda, panda, panda bear, coon bear, Ailuropoda melanoleuca ; probability=11.516657 `

and here is my predictions according to my configuration preprocess

```
282 tiger cat 12.07%
281 tabby, tabby cat 11.91%
285 Egyptian cat 10.97%
287 lynx, catamount 9.93%
289 snow leopard, ounce, Panthera uncia 8.35%

388 giant panda, panda, panda bear, coon bear, Ailuropoda melanoleuca 13.74%
341 hog, pig, grunter, squealer, Sus scrofa 5.79%
812 space shuttle 5.73%
294 brown bear, bruin, Ursus arctos 5.73%
805 soccer ball 5.64%
```

**Comment by xadupre** [comment]
Is it possible to know how you quantized your model? (which command line).

**Reply by Apisteftos** [reply]
Oh I did that exporting from pytorch to onnx, I am not sure, because it's a long time, but it seems it didn't had any good training or quantization.
